src/GNDStk/Tree.hpp

```diff
@@ -0,0 +1,97 @@
+
+#include "GNDStk/Tree/src/detail.hpp"
+
+// -----------------------------------------------------------------------------
+// Tree
+// -----------------------------------------------------------------------------
+
+template<
+   template<class...> class METADATA_CONTAINER, // container type for metadata
+   template<class...> class CHILDREN_CONTAINER  // container type for children
+>
+class Tree : public Node<METADATA_CONTAINER,CHILDREN_CONTAINER> {
```

Comment:
Ok, now I can see the updated code. I don't know what I was looking at before.

Reply:
In my opinion, Github has a cluttered and often unintuitive interface.